addons/website/views/snippets/s_closer_look.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_closer_look" name="Closer Look">
+    <section class="s_closer_look pt96">
+        <div class="container">
+            <h1 class="display-3" style="text-align: center;">Take a closer look now</h1>
+            <p class="lead" style="text-align: center;">Write one or two paragraphs describing your product, services or a specific feature.<br/> To be successful your content needs to be useful to your readers.</p>
+            <p style="text-align: center;">
+                <a t-att-href="cta_btn_href" class="btn btn-primary"><t t-esc="cta_btn_text">Our store</t></a>
+                <a t-att-href="cta_btn_href" class="btn btn-outline-primary"><t t-esc="cta_btn_text">Contact us</t></a>
```

Comment:
`t-esc="cta_btn_text"` -> these two buttons will have the same text at the end 😉
Replace `Our store` with something generic not relate to ecommerce